### PR TITLE
[Feature](Job)Support manual and refactor some execution logic

### PR DESCRIPTION
**Pull request by CalvinKirs**
Supports manually triggered JOBs and Tasks
Optimize JOB&TASK display logic
Refactor the executor to support context passing

## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by CalvinKirs** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/scheduler/job/JobTask.java [hunk 1]
```diff
@@ -54,17 +55,35 @@ public class JobTask implements Writable {
...
+    private TaskType taskType = TaskType.SCHEDULER_JOB_TASK;
+
+    /**
+     * Some parameters specific to the current task that need to be used to execute the task
+     * eg: sql task, sql it's: select * from table where id = 1 order by id desc limit ${limit} offset ${offset}
+     * contextData is a map, key1 is limit, value is 10,key2 is offset, value is 1
+     * when execute the task, we will replace the ${limit} to 10, ${offset} to 1
+     * so to execute sql is: select * from table where id = 1 order by id desc limit 10 offset 1.
+     */
+    private T contextData;
```
Comment (zddr) [inline fe/fe-core/src/main/java/org/apache/doris/scheduler/job/JobTask.java:68]: Why not persist？

Comment (CalvinKirs) [reply fe/fe-core/src/main/java/org/apache/doris/scheduler/job/JobTask.java:73]: done

**Comment by CalvinKirs** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/scheduler/executor/JobExecutor.java [hunk 2]
```diff
@@ -28,18 +28,19 @@
  * We use Gson to serialize and deserialize JobExecutor. so the implementation of JobExecutor needs to be serializable.
  * You can see @org.apache.doris.persist.gson.GsonUtils.java for details.When you implement JobExecutor,pls make sure
  * you can serialize and deserialize it.
- *
- * @param <T> The result type of the event job execution.
  */
 @FunctionalInterface
-public interface JobExecutor<T> {
+public interface JobExecutor<T, C> {
```
[the remark this answers is not in the document]
Comment (CalvinKirs) [reply fe/fe-core/src/main/java/org/apache/doris/scheduler/executor/JobExecutor.java:33]: Thx,  I will create a PR to fix it individually, and the corresponding test case

**Review by morningman — APPROVED** [review]
LGTM

**Review by zddr — APPROVED** [review]

**Pull request merged**